test/verify/check-machines

```diff
@@ -143,6 +143,12 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         else:
             b.wait_not_present(network_row)
 
+    def tearDown(self):
```

Comment:
Thanks for adding that extra debugging! However, it's imperative to call `super.tearDown()` here, so that the actual cleanup happens.

Reply:
Not literally "here", but after opening the details of course.

Reply:
So even that the selector is correct it does not really seem to expand the alert description,
https://logs.cockpit-project.org/logs/pull-14069-20200511-145631-350483cf-rhel-8-3/TestMachines-testConfigureBeforeInstall-rhel-8-3-127.0.0.2-2201-FAIL.png

It's definitely the right selector - even verified the html file from the logs.

Reply:
Well, screenshot is called before tearDown is called. You would need to do this in `addCleanup()`

Reply:
Removed that commit since it did not work with addCleanup either.
I will debug and post a separate PR for this.